packages/smooth_app/lib/pages/onboarding/country_selector.dart

```diff
@@ -40,8 +40,8 @@ class _CountrySelectorState extends State<CountrySelector> {
   @override
   void initState() {
     super.initState();
-    final UserPreferences userPreferences = context.read<UserPreferences>();
-    _initFuture = _init(userPreferences.appLanguageCode!);
+    // final UserPreferences userPreferences = context.read<UserPreferences>();
+    // _initFuture = _init(userPreferences.appLanguageCode!);
   }
 
   Future<void> _init(final String languageCode) async {
```

Comment:
Just semantic but its not called in the initState anymore

```suggestion
  Future<void> _loadLocalizedCountryNames(final String languageCode) async {
```


Would make sense to also rename the 

`_initFuture` to  `_countryNamesFuture`